keyboards/mothdotmonster/vegas/readme.md

```diff
@@ -0,0 +1,15 @@
+# VEGAS
+
+![keyboard](https://i.imgur.com/VFN43xa.jpg)
+
+Point of sale inspired keyboard with a unique layout.
+
+* Keyboard Maintainer: mothdotmonster
+* Hardware Supported: Pro Micro and compatibles
+* Hardware Availability: [shop.moth.monster](http://shop.moth.monster/product/vegas)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware?  This is needed, as a new user may not be aware of how to enter to bootloader to flash new firmware.

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```